highcharts-api/highcharts/1-simple-column/main.js

```diff
@@ -1,14 +1,13 @@
-const generateRandomNumber = () => Math.floor(Math.random() * 10);
-const generateRandomTriplet = () => Array.from({ length: 3 }, generateRandomNumber);
+const getRandomData = (length = 3) => Array.from({ length }, () => Math.floor(Math.random() * 10));
```

Comment:
With the parameter, I meant to pass it when executing the function, what you did here was to set the default value.
```suggestion
const getRandomData = (length) => Array.from({ length }, () => Math.floor(Math.random() * 10));
```
```JS
getRandomData(3);
```